package.json

```diff
@@ -13,12 +13,12 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "npm-run-all build:grunt build:handlebars",
```

Comment:
`npm-run-all` without `-c` will abort as soon as a script fails (with a non-0 exit code). `-c` will run all scripts no matter what, but in the end exit with a non-0 code if one or more scripts failed.